docs/source/flow_setup.rst

```diff
@@ -169,15 +170,34 @@ The latter command should return an output similar to:
 Copy the path up until right before /bin (i.e. /path/to/envs/aimsun_flow) and
 place it under the `AIMSUN_SITEPACKAGES` variable in flow/config.py.
 
+Finally, all Aimsun run scripts available within the Flow framework relies on
+an Aimsun template located within flow/utils/aimsun/Aimsun_Flow.ang when
+generating new networks. This template is also responsible for interacting the
+API during simulation. To ensure that it can locate the API, you will need to
+add the location to the API within the template. In order to do so:
+
+* Click on "SCENARIOS" |rarr| "Dynamic Scenario 866" in the right panel.
```

Comment:
I guess we don't need this any more. I added it in the generator file

Reply:
Oh sweet, I'll delete it then